packages/common/voter.ts

```diff
@@ -28,3 +28,14 @@ export type RegistrationStatus =
   | 'Incomplete'
   | 'Purged'
   | null
```

Comment:
When can this return `null`?  Shouldn't it always return something or it should be an 'Error'?

Reply:
According to [Alloy's API](https://docs.alloy.us/api/#/paths/~1v1~1verify/get):
```
In addition to given inputs, the API will return back registration status and registration date. If both 
registration status and registration date are null, then the person is not found on the voter files and
is therefore likely not registered. If registration status is null but there is a non-null registration
date, then the person is found on the voter files, but a voter status was not provided by the 
Secretary of State.
```
Which was why `Base.tsx` used to default `null` to `Not Found`

Reply:
Can we use 'Not Found' instead of `null`?  It would like these statuses to be human-readable.  We should transform it to `Not Found` immediately after our call to  the alloy API.